helm_deploy/prison-offender-events/templates/_envs.tpl

```diff
@@ -88,76 +88,76 @@ env:
         name: dps-rds-instance-output
         key: rds_instance_endpoint
 
-  - name: SNS_AWS_ACCESS_KEY_ID
+  - name: HMPPS_SQS_TOPICS_PRISON_EVENT_TOPIC_ACCESS_KEY_ID
     valueFrom:
       secretKeyRef:
         name: offender-events-topic
         key: access_key_id
 
-  - name: SNS_AWS_SECRET_ACCESS_KEY
+  - name: HMPPS_SQS_TOPICS_PRISON_EVENT_TOPIC_SECRET_ACCESS_KEY
     valueFrom:
       secretKeyRef:
         name: offender-events-topic
         key: secret_access_key
 
-  - name: SNS_TOPIC_ARN
+  - name: HMPPS_SQS_TOPICS_PRISON_EVENT_TOPIC_ARN
     valueFrom:
       secretKeyRef:
         name: offender-events-topic
         key: topic_arn
 
-  - name: HMPPS_SNS_AWS_ACCESS_KEY_ID
+  - name: HMPPS_SQS_TOPICS_HMPPS_EVENT_ACCESS_KEY_ID
```

Comment:
Needs to be `HMPPS_SQS_TOPICS_HMPPS_EVENT_QUEUE_ACCESS_KEY_ID` - I bet you just copied the exact same mistake I made in POSI so that one's my bad  :smile: